downstream/modules/platform/proc-benchmark-postgresql.adoc

```diff
@@ -0,0 +1,68 @@
+[id="benchmark-postgresql"]
+
+= Benchmarking storage performance for the {PlatformNameShort} PostgreSQL database
+
+The following procedure describes how to benchmark the write/read IOPS performance of the storage system to check whether the minimum {PlatformNameShort} PostgreSQL database requirements are met.
+
+.Prerequisites
+ * You have installed the Flexible I/O Tester (fio) storage performance benchmarking tool. 
++
+To install fio, run the following command as the root user:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+# yum -y install fio
+----
+
+* You have adequate disk space to store the fio test data log files.
++
+The examples shown in the procedure below require at least 60GB disk space in the `/tmp` directory:
```

Comment:
I'm just wondering for accessibility instead of "below", should we use "in the following procedure..."?